bitrise.yml

```diff
@@ -100,6 +100,7 @@ workflows:
         - content: |
             #!/bin/bash
             set -ex
+            go get -u github.com/bitrise-io/go-utils/log
```

Comment:
can you please add it with `dep` instead? that's the package manager we use at the moment with workflow editor. you can find more info here: https://golang.github.io/dep/docs/daily-dep.html#adding-a-new-dependency